source/gui/__init__.py

```diff
@@ -679,11 +694,13 @@ def __init__(self, parent):
 		self.actionsList.SetFocus()
 		self.CentreOnScreen()
 
+	def onSelectionChange(self, evt):
+		if self.actionsList.GetSelection() == 1:
+			self.restartGroupBox.Enable(True)
+		elif self.actionsList.GetSelection() != 1:
+			self.restartGroupBox.Enable(False)

```

Comment:
A more concise way to write it, with the '1' value present only at one place. If however you find a way to get rid of this '1' hard-coded value, it's still better!

```suggestion
		self.restartGroupBox.Enable(self.actionsList.GetSelection() == 1)
```